packages/block-library/src/search/style.scss

```diff
@@ -67,7 +68,7 @@
 
 	// For lower specificity.
 	:where(.wp-block-search__button) {
-		padding: 0.125em 0.5em;
+		padding: 0.125rem 0.5rem;
```

Comment:
Another case for the `$grid-unit-x` vars. The need for non-standard padding and margin values does limit the benefit though.